### ISSUE TITLE
The prisma migrate command should have an option to start from scratch 

### ISSUE THREAD

**Issue by beeman** [prisma/migrate#436]
## Problem

Currently there seems to be no way to clear the `_Migration` table from the command line. 

If a user want to start with their migrations from scratch, this prevents them from simply nuking `prisma/migrations` and doing a new `prisma migrate save`.

The error message that we get is:

`Error: There are more migrations in the database than locally. This must not happen. Local migration ids: . Remote migration ids: <some-migration>`

## Solution

If there was a way to clean up the `_Migrations` table, the development flow would be way simpler where we could basically do a reset and start from scratch.

I understand that this is not a very common requirement with 'living' applications, where keeping migrations might be very desired. However, when working on new projects, starter kits or other code that the dev wants to iterate on, it can be really helpful to quickly start from a blank slate, and make your project forget about any previous attempts. Keeping the migration history there does probably more damage than good :)

## Alternatives

I would love it for the whole migration system to be optional, have `schema.prisma` be the source of truth similar to how it worked in Prisma 1. A developer should be able to toggle migrations in their `schema.prisma`, and from that moment on it's taken into account. With the toggle disabled it transparently always re-applies the schema as if it was clean.

## Additional context

In order to get my desired workflow I just created a simple script that nukes the `_Migrations` table, it might be useful for others:

```typescript
import { config } from 'dotenv'
import { Client } from 'pg'

config()

async function main() {
  const url = process.env.DATABASE_URL
  const schema = url.split('=')[1]
  const client = new Client(url)
  client.connect()
  console.log(' Removing migrations table', url)
  await client.query(`truncate table "${schema}"."_Migration"`)
  client.end()
}

main()
```

**Comment by beeman** [comment]
I'm looking at the docs and found [the section on troubleshooting](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-migrate#troubleshooting) that mentions deleting the migrations folder and truncating the `_Migrations` table, which is basically the 'feature' I asked for in this request.

I would be more than happy (and have some time) to give a shot at implementing this, but it would be helpful if somebody who knows the codebase could give me a high-level idea of where I would start, and what to take into account.

**Comment by matthewmueller** [comment]
I ran into this as well. I think we should definitely support something more actionable here. Right now you're left in the position, "well now what..."

At the very least, we should provide more actionable advice on how to proceed (e.g. you'll need to clear your _Migrations table), with maybe a URL with more information on how to get out of this state.

**Comment by beeman** [comment]
@matthewmueller thanks for chiming in! In the meanwhile I've created a script locally to get me this workflow, and I'm generally happy about that workflow. 

However, my script is not flawless and I think it would already be a great start if there would be a way to truncate the `_Migrations` table through the CLI. 

The main reasons that this is hard to get right in my custom script, is that I have to figure out the DB connection string/type and, depending on that, connect to the DB and execute the SQL command. This is something the Prisma CLI already does very well.

To give an example, somehow my script does not connect properly to my managed Postgres instance on DigitalOcean, but the Prisma migrate command does do it properly. 

Deleting the local migrations folder is pretty trivial and might not always be desired as it's a source of truth, so I would say it's of less importance.

**Comment by albertoperdomo** [comment]
@beeman I believe with Prisma Migrate Preview ([released in 2.13.0](https://github.com/prisma/prisma/releases/tag/2.13.0)) and prisma db push ([released in Preview in 2.10.0](https://github.com/prisma/prisma/releases/tag/2.10.0)) this issue should no longer relevant.

If it still is, please open a new issue [here](https://github.com/prisma/prisma/issues/new/choose)).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue]([https://github.com/prisma/prisma/issues/4531](https://github.com/prisma/prisma/issues/4531)).
